docs/fsharp/language-reference/import-declarations-the-open-keyword.md

```diff
@@ -46,19 +43,19 @@ Some namespaces are so frequently used in F# code that they are opened implicitl
 
 |Namespace|Description|
 |---------|-----------|
-|`Microsoft.FSharp.Core`|Contains basic F# type definitions for built-in types such as `int` and `float`.|
-|`Microsoft.FSharp.Core.Operators`|Contains basic arithmetic operations such as `+` and `*`.|
-|`Microsoft.FSharp.Collections`|Contains immutable collection classes such as `List` and `Array`.|
-|`Microsoft.FSharp.Control`|Contains types for control constructs such as lazy evaluation and asynchronous workflows.|
-|`Microsoft.FSharp.Text`|Contains functions for formatted IO, such as the `printf` function.|
+|`FSharp.Core`|Contains basic F# type definitions for built-in types such as `int` and `float`.|
```

Comment:
holy breaking change, Batman!.  :bat: really?

Reply:
Oh, this one will tickle you. These namespaces are actually all auto-opened anyways, so the vast majority of code will never notice anything.

But it gets better. The F# compiler actually treats these namespaces as equivalent even if they're explicitly opened. It's kind of gross, but ... it is what it is, and does let us drop the prefix.